projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -83,86 +119,52 @@ public static void saveCalendar(Path csvPath, String path) {
-            .setThursday(Integer.parseInt(record.get("thursday")))
-            .setFriday(Integer.parseInt(record.get("friday")))
-            .setSaturday(Integer.parseInt(record.get("saturday")))
-            .setStartDate(Integer.parseInt(record.get("start_date")))
-            .setEndDate(Integer.parseInt(record.get("end_date")))
-            .build()
-            .writeDelimitedTo(output);
+        if (isServiceIdRelevantToday(record)) System.out.println(record.toString());
       }
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
 
-  public static void saveFareAttributes(Path csvPath, String path) {
+  public static void saveCalendar(Path csvPath, String path) {
     try (FileOutputStream output = new FileOutputStream(path)) {
       CSVParser parser =
           CSVParser.parse(
               csvPath,
               StandardCharsets.UTF_8,
               CSVFormat.DEFAULT.withHeader(
-                  "fare_id",
-                  "price",
-                  "currency_type",
-                  "payment_method",
-                  "transfers",
-                  "agency_id",
-                  "transfer_duration"));
+                  "service_id",
+                  "sunday",
+                  "monday",
+                  "tuesday",
+                  "wednesday",
+                  "thursday",
+                  "friday",
+                  "saturday",
+                  "start_date",
+                  "end_date"));
       boolean isHeader = true;
       for (CSVRecord record : parser) {
         if (isHeader) {
           isHeader = false;
           continue;
         }
-        FareAttributes.Builder fareAttributes = FareAttributes.newBuilder();
-        fareAttributes
-            .setFareId(Integer.parseInt(record.get("fare_id")))
-            .setPrice(Double.parseDouble(record.get("price")))
-            .setCurrencyType(record.get("currency_type"))
-            .setPaymentMethod(Integer.parseInt(record.get("payment_method")))
-            .setTransfers(Integer.parseInt(record.get("transfers")))
-            .setAgencyId(Integer.parseInt(record.get("agency_id")))
-            .setTransferDuration(record.get("transfer_duration"))
-            .build()
-            .writeDelimitedTo(output);
-      }
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
-  }
-
-  public static void saveFareRules(Path csvPath, String path) {
-    try (FileOutputStream output = new FileOutputStream(path)) {
-      CSVParser parser =
-          CSVParser.parse(
-              csvPath,
-              StandardCharsets.UTF_8,
-              CSVFormat.DEFAULT.withHeader(
-                  "fare_id", "route_id", "origin_id", "destination_id", "contains_id"));
-      boolean isHeader = true;
-      for (CSVRecord record : parser) {
-        if (isHeader) {
-          isHeader = false;
-          continue;
+        if (isServiceIdRelevantToday(record)) {
```

Comment:
I don't think we need to save the calendar CSV, just filter by it.

Reply:
Do you mean we should not serialize to a file the filtered messages of any CSVs?

Reply:
Calendar specifically is used to filter the trips, but doesn't otherwise contain useful information.